apps/baseline/serializers.py

```diff
@@ -1466,3 +1470,175 @@ def get_strategy_label(self, obj):
+        "strategy_type": "livelihood_strategies__strategy_type__iexact",
+    }
+
+    livelihood_zone_name = DictQuerySetField("livelihood_zone_name")
+    source_organization_name = DictQuerySetField("source_organization_pk")
+    country_pk = DictQuerySetField("country_pk")
+    country_iso_en_name = DictQuerySetField("country_iso_en_name")
+    livelihoodzone_pk = DictQuerySetField("livelihoodzone_pk")
+    livelihood_strategy_pk = DictQuerySetField("livelihood_strategy_pk")
+    livelihood_activity_pk = DictQuerySetField("livelihood_activity_pk")
+    wealth_group_category_code = DictQuerySetField("wealth_group_category_code")
+    id = DictQuerySetField("id")
+    name = DictQuerySetField("name")
+    description = DictQuerySetField("description")
+    source_organization = DictQuerySetField("source_organization")
+    livelihood_zone = DictQuerySetField("livelihood_zone")
+    main_livelihood_category = DictQuerySetField("main_livelihood_category")
+    bss = DictQuerySetField("bss")
+    currency = DictQuerySetField("currency")
+    reference_year_start_date = DictQuerySetField("reference_year_start_date")
+    reference_year_end_date = DictQuerySetField("reference_year_end_date")
+    valid_from_date = DictQuerySetField("valid_from_date")
+    valid_to_date = DictQuerySetField("valid_to_date")
+    population_source = DictQuerySetField("population_source")
+    population_estimate = DictQuerySetField("population_estimate")
+    product_cpc = DictQuerySetField("product_cpc")
+    product_common_name = DictQuerySetField("product_common_name")
+    strategy_type = DictQuerySetField("strategy_type")
+
+    slice_sum_kcals_consumed = DictQuerySetField("slice_sum_kcals_consumed")
+    sum_kcals_consumed = DictQuerySetField("sum_kcals_consumed")
+    kcals_consumed_percent = DictQuerySetField("kcals_consumed_percent")
+
+    def get_fields(self):
+        """
+        User can specify fields= parameter to specify a field list, comma-delimited.
+
+        If the fields parameter is not passed or does not match fields, defaults to self.Meta.fields.
+
+        The aggregated fields self.aggregates are added regardless of user field selection.
+        """
+        field_list = "request" in self.context and self.context["request"].query_params.get("fields", None)
+        if not field_list:
+            return super().get_fields()
+
+        # User-provided list of fields
+        field_names = set(field_list.split(","))
+
+        # Add the aggregates that are always returned
+        for field_name, aggregate in self.aggregates.items():
+            field_names |= {
+                field_name,
+                self.aggregate_field_name(field_name, aggregate),
+                self.slice_aggregate_field_name(field_name, aggregate),
+                self.slice_percent_field_name(field_name, aggregate),
+            }
+
+        # Add the ordering field if specified
+        ordering = self.context["request"].query_params.get("ordering")
```

Comment:
If we are hard-coding the ordering parameter rather than using https://www.django-rest-framework.org/api-guide/settings/#ordering_param we need to document it. It would be better to use `settings.ORDERING_PARAM`